mpyl_stages.schema.yml

```diff
@@ -0,0 +1,25 @@
+$schema: http://json-schema.org/draft-07/schema#
```

Comment:
I would move this file to the schema-s folder in `src/mpyl/schema`

Reply:
The schema of the stages needs to be adjustable, so we can't put it in `src/mpyl/schema`

Reply:
I fixed the health error here https://github.com/Vandebron/mpyl/pull/333/commits/b73ade4786c2d47df15ac04bd900672337c14a25